akka-actor-tests/src/test/scala/akka/actor/ActorWithStashSpec.scala

```diff
@@ -15,7 +15,7 @@ import com.github.ghik.silencer.silent
 
 import scala.concurrent.duration._
 import org.scalatest.BeforeAndAfterEach
-import org.scalatestplus.junit.JUnitSuiteLike
+import org.scalatest.junit.JUnitSuiteLike
```

Comment:
We ran up against an issue with this several months ago when we could not make this change yet. We're sure it's ok here now?

Reply:
This specific instance is OK since it's in `akka-actor-tests`.

The dependency update might indeed be problematic though - will comment there.

I don't remember why